vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -35,6 +35,32 @@ function DownloadFileWithRetry {
     }
 }
 
+function Download-FileWithAzCopy {
+    param (
+        $URL,
+        $Dest
+    )
+
+    if (!(Test-Path -Path "c:\azcopy")) {
```

Comment:
Please define a global variable `C:\akstemp` and use it in all places.
You can use `C:\akstemp\azcopy` as the folder for azcopy